src/kaleidoscope/plugin/LEDEffect-Rainbow.cpp

```diff
@@ -63,7 +63,11 @@ void LEDRainbowWaveEffect::TransientLEDMode::update(void) {
 
   for (auto key_addr : KeyAddr::all()) {
     uint16_t key_hue = rainbow_hue + 16 * (key_addr.toInt() / 4);
-    if (key_hue >= 255)          {
+    // This is required to support more than 128 LEDs.
+    if (key_hue >= 511) {
```

Comment:
I feel like this comment doesn't really explain what's going on, especially in the context of the existing code with the 255 decrement.

Reply:
Indeed, it doesn't. I remade the PR, with a different solution that'll work for more than 256 LEDs too, and a much more explanatory comment added (which is mirrored in the commit message too).